app/konnect/service-catalog/integrations/github.md

```diff
@@ -32,6 +32,17 @@ Entity  | Description
 -------|-------------
 Repository | A GitHub repository relating to the service
 
+## Events
+
+The Eventing feature supports viewing the following event types for linked repositories from the {{site.konnect_product_name}} UI.
+
+| Event Type                 |
+|----------------------------|
+| Open pull request       |
+| Merge pull request       |
+| Close pull request         |
```

Comment:
Is this table unfinished, or meant to only have one column? If it's meant to look like this, then I'd recommend turning it into a list instead.